ProfessorAssessor/View Controllers/HomeViewController.m

```diff
@@ -16,14 +17,22 @@ - (void)viewDidLoad {
 }
 
 - (IBAction)logout:(UIBarButtonItem *)sender {
-    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
-        UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-        LoginViewController *loginViewController = [storyboard instantiateViewControllerWithIdentifier:@"LoginViewController"];
+    [PFUser logOutInBackgroundWithBlock:^(NSError *_Nullable error) {
+        if (FBSDKAccessToken.currentAccessTokenIsActive) {
+            [[FBSDKLoginManager alloc] logOut];
```

Comment:
You can double check but I'm 90% sure this method can be called safely regardless of accessToken and will also clear the AuthToken. Curious why you did this way... if it's documentation I can change it so other people don't make the same mistake. :)